omnigan/eval_metrics.py

```diff
@@ -0,0 +1,66 @@
+from PIL import Image
```

Comment:
Oh it did that for you too ! wonder why an empty file was pushed in the first place ...